storage/statedb/cache.go

```diff
@@ -19,17 +19,21 @@ package statedb
 import (
 	"errors"
 	"strings"
+
+	"github.com/go-redis/redis/v7"
 )
 
 type TrieNodeCacheType string
 
 // TrieNodeCacheConfig contains configuration values of all TrieNodeCache.
 type TrieNodeCacheConfig struct {
-	CacheType          TrieNodeCacheType
-	LocalCacheSizeMB   int      // Memory allowance (MB) to use for caching trie nodes in fast cache
-	FastCacheFileDir   string   // Directory where the persistent fastcache data is stored
-	RedisEndpoints     []string // Endpoints of redis cache
-	RedisClusterEnable bool     // Enable cluster-enabled mode of redis cache
+	CacheType           TrieNodeCacheType
+	LocalCacheSizeMB    int      // Memory allowance (MB) to use for caching trie nodes in fast cache
+	FastCacheFileDir    string   // Directory where the persistent fastcache data is stored
+	RedisEndpoints      []string // Endpoints of redis cache
+	RedisClusterEnable  bool     // Enable cluster-enabled mode of redis cache
+	RedisPublishBlock   bool     // Enable publishing every inserted block to the redis server
+	RedisSubscribeBlock bool     // Enable subscribing blocks from the redis server
```

Comment:
```suggestion
	RedisPublishBlockEnable   bool     // Enable publishing every inserted block to the redis server
	RedisSubscribeBlockEnable bool     // Enable subscribing blocks from the redis server
```
The names sounds like a block number of recent redis subscription.

Reply:
The new commit took your suggestion. Thanks!